NuGet.config

```diff
@@ -7,8 +7,7 @@
     <add key="aspnet-extensions" value="https://dotnetfeed.blob.core.windows.net/aspnet-extensions/index.json" />
     <add key="myget-roslyn" value="https://dotnet.myget.org/F/roslyn/api/v3/index.json" />
     <add key="myget-roslyn-tools" value="https://dotnet.myget.org/F/roslyn-tools/api/v3/index.json" />
-    <add key="myget-vssdk" value="https://vside.myget.org/F/vssdk/api/v3/index.json" />
-    <add key="myget-vsmac" value="https://vside.myget.org/F/vsmac/api/v3/index.json" />
```

Comment:
Interesting, didn't think we'd be able to remove the vsmac feed. If it builds, :shipit:

Reply:
It's been merged into the same feed as devcore